docs-site/content/.vuepress/config.js

```diff
@@ -149,6 +149,12 @@ let config = {
             ['/0.19.0/guide/features/typo-tolerance', 'Typo Tolerance'],
```

Comment:
What I run this on dev, I see this error: 

```
No matching page found for sidebar item "/0.19.0/guide/features/curation"
```

I don't see a file called curation.md in the repo

Reply:
I haven't added it yet. Will be added in future PR. Do you want me to add a placeholder file?

Reply:
Ah I see, in that case let's remove the line from the side-bar configuration, so the nav renders in the meantime.